index.js

```diff
@@ -113,7 +113,7 @@ function archiveResponse (datUrl, archive, req, res) {
 
 function onerror (status, res) {
   if (typeof status !== 'number') {
-    console.error(status)
+    // console.error(status)
```

Comment:
Oh ya this was a placeholder. We can remove it, any idea of better errors besides just giving a 404? Not sure if this is something we want to show the user though.